### ISSUE TITLE
have you ever try put weights on loss function?

### ISSUE THREAD

**Issue by LiuHao-THU** [shun1024/weighted-deepwalk#7]
loss = pos_loss * weights + neg_loss
